qp_klp/tests/test_step.py

```diff
@@ -271,6 +272,24 @@ class BaseStepTests(TestCase):
                 "job_pool_size": 30,
                 "job_max_array_length": 1000
             },
+            "nu-qc": {
+                "nodes": 1,
+                "nprocs": 16,
```

Comment:
Based on the other open PR, should this be connected/the-same as {{cores_per_task}}? If yes, could you make sure to use the same name, just for clarity?

Reply:
Good catch! I removed nprocs for NuQC because it's not being used and added cpus_per_task since it is being used. I added a comment briefly covering the difference between the two parameters after reading some literature.